apis/flowcollector/v1beta1/flowcollector_types.go

```diff
@@ -325,6 +327,7 @@ type FlowCollectorEBPF struct {
 	// If the `spec.agent.ebpf.privileged` parameter is not set, an error is reported.<br>
 	// - `DNSTracking`: enable the DNS tracking feature.<br>
 	// - `FlowRTT`: enable flow latency (sRTT) extraction in the eBPF agent from TCP traffic.<br>
+	// - `NetworkEventsMonitor`: enable the Network events monitoring feature.<br>
```

Comment:
like in PacketDrop, we should mention here if Privileged is required
Also, maybe add more details right here because it's not super obvious what are network events. like ".. such as network policy verdicts" ?